server/chat-plugins/mafia.ts

```
@@ -2480,10 +2488,13 @@ export const commands: Chat.ChatCommands = {
 			const game = this.requireGame(Mafia);
 			if (game.hostid !== user.id && !game.cohostids.includes(user.id)) this.checkCan('mute', null, room);
 			if (target) return this.parse('/help mafia resetgame');
-			if (game.phase !== 'day' && game.phase !== 'night') throw new Chat.ErrorMessage(`The game has not started yet.`);
-			if (game.IDEA.data) throw new Chat.ErrorMessage(`You cannot use this command in IDEA.`);
-			game.resetGame();
-			game.logAction(user, 'reset the game state');
+			if (game.phase !== 'day' && game.phase !== 'night') return this.errorReply(`The game has not started yet.`);
+			if (game.IDEA.data) return this.errorReply(`You cannot use this command in IDEA.`);
+			try {
+				game.resetGame(user);
+				game.logAction(user, 'reset the game state');
+				// try-catch block is only so game.logAction can run.If this becomes a non issue at some point,this should be refactored.
+			} catch {}
```

Typically its bad practice to suppress errors like this.